src/lib/flow/sol-flow-resolver-conffile.c

```diff
@@ -112,6 +119,36 @@ get_module_for_type(const char *type)
     return SOL_STR_SLICE_STR(type, sep -  type);
 }
 
+
+static int
+find_entry_by_match(struct sol_vector *entries,
+    const struct resolver_conffile_dlopen *match)
+{
+    struct resolver_conffile_dlopen *entry;
+    int i;
+
+    SOL_VECTOR_FOREACH_IDX (entries, entry, i) {
+        if (entry == match)
+            return i;
+    }
+
+    return -1;
+}
+
+static struct resolver_conffile_dlopen *
+find_entry_by_name(struct sol_vector *entries, const char *name)
+{
+    struct resolver_conffile_dlopen *entry;
+    int i;
```

Comment:
If I make 'entry' const in this context I will need to drop its constness, I think it's better to leave this as it is.